scene/main/viewport.cpp

```diff
@@ -4105,6 +4105,16 @@ TypedArray<Window> Viewport::get_embedded_subwindows() const {
 	return windows;
 }
 
+void Viewport::set_drag_threshold(int p_threshold) {
+	ERR_MAIN_THREAD_GUARD
+	gui.drag_threshold = p_threshold;
+}
+
+int Viewport::get_drag_threshold() const {
+	ERR_READ_THREAD_GUARD_V(false);
```

Comment:
```suggestion
	ERR_READ_THREAD_GUARD_V(10);
```

This is a return value, so should be same type (either default or 0).